rskj-core/src/main/java/co/rsk/peg/federation/FederationMember.java

```diff
@@ -37,6 +39,11 @@
  */
 public final class FederationMember {
     public static final FederationMemberPubKeysComparator BTC_RSK_MST_PUBKEYS_COMPARATOR = new FederationMemberPubKeysComparator();
+    private static final int FEDERATION_RLP_LIST_SIZE = 3;
+    private static final int LIST_SIZE = 3;
```

Comment:
Pretty sure these 2 represent the same thing. And I'd try to think of a better name, KEYS_COUNT maybe or something of the sort